src/boot/efi/cpio.c

```diff
@@ -486,7 +486,7 @@ EFI_STATUS pack_cpio(
         *ret_buffer = NULL;
         *ret_buffer_size = 0;
 
-        if (ret_measured)
+        if (ret_measured && n_tpm_pcr > 0)
                 *ret_measured = true;
```

Comment:
Hmm, doesn't this violate the rule that the output variable should be always set?

Reply:
yes, i am an idiot ;-)